### DISCUSSION TITLE
Select book reading order and pages.

### DISCUSSION THREAD

**Discussion by jongfeel** [ThinkAboutSoftware/AcademicConference#5] [General]
곧 모집이 마감입니다.

아래 세 권의 책이 선정 되었는데요.

- [소프트웨어 장인](http://aladin.kr/p/TGFJc)
- [클린 코더](http://aladin.kr/p/MJJw1)
- [소프트웨어 스펙의 모든 것](http://aladin.kr/p/WM510)

책 읽는 순서와 각 책 별로 읽을 범위를 결정해야 합니다.

그냥 정하자고 하면 안정해지기 때문에 제가 일단 정해보고 의견을 나눠보면 좋을 것 같습니다.

- 4월, 소프트웨어 스펙의 모든 것
  - 4월 16일: 1, 2, 3장 약 50 페이지
  - 4월 30일: 4, 5장 약 46 페이지
- 5월, 클린 코더
  - 5월 14일: 1, 2, 3장 약 45 페이지
  - 5월 28일; 4, 5, 6장 약 42 페이지
- 6월, 소프트웨어 장인
  - 6월 11일: 1, 2, 3, 4장 약 74 페이지
  - 6월 25일: 5, 6, 7, 8장 약 99 페이지

소프트웨어 장인이 조금 분량이 많은데, 욕심 내서 1부 까지 읽어 보는게 좋을 것 같아서 일부러 정해봤습니다.

세부 규칙은 아래와 같습니다.

- 약 10일간 정해진 분량의 책을 읽고, 리뷰 D-3일 전까지 논의할 내용 pull request로 등록
- 리뷰 D-Day 까지 각 멤버별로 등록한 pull request에 코멘트를 달면서 논의 진행 완료
- 리뷰 D-Day에 온라인으로 실제 얘기하고 싶었던 부분에 대해 짧게 얘기하며 진행
- 학술 회의날 까지 총 5회 반복, 마지막 6회는 학술회의날 진행

이렇게 입니다.

**Comment by Joseph-Cha** [comment]
리뷰 D-3일 전까지 논의할 내용 pull request를 등록해야하니깐 
정해주신 날짜에서 3일 전까지(ex. 소프트웨어 스펙의 모든 것 1,2,3장은 4월 13일까지) pull request를 날리면 된다는거군요?!

개인적으로 분량은 합리적이라고 생각합니다~
보통 2시간에 20페이지 정도를 소화하는 것 같아서 2주 동안 약 6시간 정도를 투자하면 될 것 같네요 ㅎㅎ

**Reply by jongfeel** [reply]
맞습니다!
동훈님은 안해보셔서 적응이 안되실 수 있는데, 다른 분은 이미 빡세게(?) 해보셔서 잘 아실겁니다. ㅎㅎ

**Comment by jongfeel** [comment]
알람이 안간거 같아서 알람 주는 용도로 적어 봅니다.
(이미 코멘트 다신 동훈님 제외)

@hi-hyein 
@yoonucho

**Comment by yoonucho** [comment]
넵 알겠습니다 ! 준비해보겠습니다!

**Comment by hi-hyein** [comment]
친절하게 분량을 정해주셨는데, 그 분량 또한 완벽해서 이견 없습니다ㅋㅋㅋㅋㅋㅋㅋ
열심히 준비해 볼게요~

**Comment by jongfeel** [comment]
@Joseph-Cha 
@hi-hyein 
@yoonucho 

제가 한가지 놓친게 있습니다.

~~소프트웨어 장인 책 없으신분?~~

읽어야 할 책 목록 중에 안가지고 계신 분은 어떤 책이 없는 지 얘기해 주세요

**Reply by jongfeel** [reply]
- 소프트웨어 장인: 회사, 집에 한 권씩 있음
  - 회사에 있는 책 => 제가 읽을 예정
  - 집에 있는 책 => 동훈님 줄 예정
- 클린 코더: 회사에 한 권 있음
  - 회사에 있는 책 => 제가 읽을 예정
  - 새 책 주문 => 동훈님 드릴 예정
- 소프트웨어 스펙의 모든 것: 회사에 한 권 있음
  - 회사에 있는 책 => 동훈님
  - 새 전자책 주문 => 제가 읽을 예정

동훈님과 저의 계획은 이렇습니다.

**Reply by hi-hyein** [reply]
저는 소프트웨어 장인, 소프트웨어 스펙의 모든 것, 두 권이 없어요~

**Reply by jongfeel** [reply]
그러면 소프트웨어 스펙의 모든 것 부터 보셔야 하니 제가 ebook으로 선물해 드리겠습니다.

**Reply by yoonucho** [reply]
저는 클린코더는 아직 구입을 안했네요 ㅎㅎ

**Reply by jongfeel** [reply]
그러면 클린 코더 읽을 때 구매해서 전달 드리겠습니다.
(클린 코더는 ebook이 없는게 아쉽군요)

**Reply by yoonucho** [reply]
헉 감사합니다 요즘 야근하느라 이제 봤어요 ㅠㅠ

**Comment by hi-hyein** [comment]


**Reply by jongfeel** [reply]
민감한 문제일 수 있으므로 따로 연락 드리겠습니다!
